Improve wording on Broker configuration (#1904)
centreon/centreon-documentation: versioned_docs/version-22.04/graph-views/remote-server.md

## Centreon Broker configuration

> Configuring the Centreon MAP module on a Centreon remote server requires
> configuring the Centreon Broker from the Centreon central server so that, with a dedicated Centreon

Reviewer: ```suggestion
In order to display real time statuses, **Centreon MAP** needs to receive this data from the Centreon remote server's Broker.
```

> Broker output on the Centreon remote server, Centreon MAP can receive
> real time data directly from its Centreon remote server.

Reviewer: ```suggestion
```

> To do so, you need to modify the Centreon Broker Master configuration of the

Reviewer: ```suggestion
To do so, you need to log in to your central server and modify the Centreon Broker Master configuration of the
```
